server/api.py

```diff
@@ -286,3 +285,21 @@ def create_session(claims, token_user):
 
         return jsonify(success="Successfully created session", session=new_session), 200, json_tag
 
+    def read_session(claims, token_user, id):
+        """ Returns the session with the given id """
+        user = None
+        try:
+            user = token_user
```

Comment:
really confused why this is wrapped in a try-except, i don't think it's even possible to get an exception on an assignment